### PR TITLE
fix: [M3-8161] - Fix misaligned DNS banner text

### PR DESCRIPTION
**Pull request by hkhalil-akamai**
## Description 📝
Fixes an alignment issue in the DNS zones banner. This banner appears in the domains landing when the user has a DNS zone but no Linodes on their account.

## Preview 📷
|Breakpoint|Before|After|
|--|--|--|
|`lg`|Screenshot 2024-09-11 at 3 59 48 PM|Screenshot 2024-09-11 at 3 59 56 PM|
|`md`|Screenshot 2024-09-11 at 4 00 28 PM|Screenshot 2024-09-11 at 4 00 23 PM
|`sm`|Screenshot 2024-09-11 at 4 00 45 PM|Screenshot 2024-09-11 at 4 00 53 PM|

### CODE REVIEW

File: packages/manager/src/features/Domains/DomainBanner.tsx [hunk 1]
```diff
@@ -43,12 +43,6 @@ export const DomainBanner = React.memo((props: DomainBannerProps) => {
   );
 });
 
-const StyledTypography = styled(Typography, { label: 'StyledTypography' })(
-  ({ theme }) => ({
-    marginBottom: theme.spacing(),
```
Comment (hkhalil-akamai) [inline packages/manager/src/features/Domains/DomainBanner.tsx:48]: Unsure of the original purpose of this margin.

File: packages/manager/src/features/Domains/DomainBanner.tsx [hunk 2]
```diff
@@ -30,9 +30,9 @@ export const DomainBanner = React.memo((props: DomainBannerProps) => {
       variant="warning"
     >
       <>
-        <StyledTypography>
+        <Typography>
           <strong>Your DNS zones are not being served.</strong>
-        </StyledTypography>
+        </Typography>
```
Comment (dwiley-akamai) [inline packages/manager/src/features/Domains/DomainBanner.tsx:35]: As you shrink the window, there are widths where this text and the other piece of text get very close to each other (example below). Maybe some padding or margin can be added on the right here to ensure there's always a gap?

<details>
<summary>Example (1063px width)</summary>

Screenshot 2024-09-11 at 4 45 16 PM
</details>

**Review by dwiley-akamai — APPROVED** [review]
Alignment improvement verified ✅

File: packages/manager/src/features/Domains/DomainBanner.tsx (hunk 1 above)
Comment (pmakode-akamai) [reply packages/manager/src/features/Domains/DomainBanner.tsx:48]: Perhaps we could keep `StyledTypography` and replace `marginBottom` with `marginRight`?

**Comment by hkhalil-akamai** [comment]
@dwiley-akamai @harsh-akamai I'm experimenting with using `Stack` to layout the elements vertically. This resolves the spacing issue and looks better in my opinion. Are there any objections to the following:

|Breakpoint|Screenshot|
|-|-|
|`lg` and `md`| Screenshot 2024-09-12 at 1 51 07 PM|
|`sm`|Screenshot 2024-09-12 at 1 51 52 PM|

**Comment by pmakode-akamai** [comment]
> @dwiley-akamai @pmakode-akamai I'm experimenting with using `Stack` to layout the elements vertically. This resolves the spacing issue and looks better in my opinion.

Nice work 🚀 Looks good to me!

**Review by cpathipa — APPROVED** [review]
DNS Banner alignment verified.

**Comment by dwiley-akamai** [comment]
> @dwiley-akamai @pmakode-akamai I'm experimenting with using `Stack` to layout the elements vertically. This resolves the spacing issue and looks better in my opinion. Are there any objections to the following:

I like this much better as well 👍🏾

**Pull request merged**